src/miniscript/types/mod.rs

```diff
@@ -820,16 +821,16 @@ impl Property for Type {
                 // Note that for CLTV this is a limitation not of Bitcoin but Miniscript. The
                 // number on the stack would be a 5 bytes signed integer but Miniscript's B type
                 // only consumes 4 bytes from the stack.
-                if t == 0 || (t & SEQUENCE_LOCKTIME_DISABLE_FLAG) != 0 {
+                if t == PackedLockTime::ZERO {
```

Comment:
Did the code originally attempt to check if lock times are actually enabled or something?

Reply:
Originally both sequnece and locktime (`After` and `Older`) where handled together. When I split them apart I kept the disable check even though it seemed wrong, finally I got the confidence to remove it :)

Reply:
It's not just after vs older -- the original code would look at a `u32` which might contain a value that didn't encode any locktime at all.

It's not clear to me that this check can be removed.

Reply:
No worries, since this PR is not a merge candidate this bridge doesn't need crossing now. I imagine between @apoelstra and @sanket1729 this will have to be worked out when miniscript upgrades to use rust-bitcoin 0.29.0.

Reply:
Flagging this thread, this will need resolving before we can consider upgrading to the newly release version of `rust-bitcoin`. Sorry to be a drag but I think this is kind of subtle and will need input by @sanket1729 and/or @apoelstra to work it out. I can code up the solution.

Reply:
@apoelstra are you convinced that this is resolved?

Reply:
Let me re-read this PR.